text/0630-new-browser-support-policy.md

```diff
@@ -0,0 +1,67 @@
+- Start Date: 2020-05-21
+- Relevant Team(s): All
+- RFC PR: https://github.com/emberjs/rfcs/pull/630
+- Tracking: (leave this empty)
+
+# New Browser Support Policy
+
+## Summary
+
+> In dropping support for IE11 at some point in the future, we will need a new
+browser support policy. This is the proposed policy.
+
+## Motivation
+
+> It should be clear which browsers are supported by Ember.js
+
+## Detailed policy
+
+- Desktop
+
+    1. Google Chrome
+
+        - The current and previous stable releases are supported
+
+    2. Mozilla Firefox
+
+        - The current and previous rapid releases and any [ESR](https://support.mozilla.org/en-US/kb/choosing-firefox-update-channel) versions currently supported by Mozilla are supported
+
+    3. MS Edge
+
+        - The current and previous stable releases are supported.
+
+    4. Mac Safari
+
+        - The versions coming with the current and previous releases of MacOS are supported, as well as versions of Mac Safari available with updates to the current release.
+
+- Mobile
+
+    1. Google Chrome
+
+        - The versions coming with all versions of Android released in the past 36 months are supported, as well as the current and most stable releases available via update.
+
+    2. iOS Safari
+
+        - The versions coming with all versions of iOS released in the past 36 months are supported.
+
+    3. Android
+
+        - The versions released with all versions of Android released in the past 36 months are supported.
+
+All browsers not listed here may work but are not explicitly supported.
+
+## How we teach this
+
+> A blog post explaining the policy with a link to this RFC will be posted.
+
+## Drawbacks
+
+- See discussions on this RFC
+
+## Alternatives
+
+> See discussions on this RFC
+
+## Unresolved questions
+
+> Should a version of non Chromium MS Edge be supported, and if so for how long?
```

Comment:
```suggestion
- Should a version of non-Chromium MS Edge be supported, and if so for how long?
- What are the support schedules for the respective browser vendors? 
```

Reply:
The support schedules are known and documented, however practically we must support more browsers for a longer period of time than browsers are supported by vendors.

Reply:
> however practically we must support more browsers for a longer period of time than browsers are supported by vendors

I'm not quite sure this is true. I don't think we should support browsers that themselves are not supported. 🤔

Reply:
For example, IE11 has not been officially supported by Microsoft since 2015 except for security updates.

Reply:
> officially supported by Microsoft since 2015 except for security updates

These two parts are contradictory 😸

Security updates === supported :wink:

Reply:
In that case we may end up supporting IE11 indefinitely, as long as Windows exists.

Reply:
Not recommended for use === supported?

Also, as of January 2020, corporations and governments using IE11 are supposed to pay for those security updates.

Reply:
> > however practically we must support more browsers for a longer period of time than browsers are supported by vendors

> I'm not quite sure this is true. I don't think we should support browsers that themselves are not supported. 🤔

I'll give another example. For Chrome, only the stable and previous release are supported by Google. However, many enterprise desktops and mobile users are running much older versions of Chrome. Enterprise users are kept on older browsers to avoid frequent testing of internal apps. Mobile users often stop upgrading their browsers once phone vendors stop shipping updates, leaving a large segment with old versions.